api-tester/src/main/java/com/revenuecat/apitester/kotlin/PackageAPI.kt

```diff
@@ -11,7 +11,7 @@ private class PackageAPI {
             val identifier: String = identifier
             val packageType: PackageType = packageType
             val product: StoreProduct = product
-            val offering: String = offering // TODOBC5 change to offering?
+            val offering: String = offering
```

Comment:
we were thinking we'd either pass the full offering or rename to offeringId, but confirmed w cesar that this probably isn't worth it...

Reply:
Hmm might be good deprecating and renaming to `offeringId`, since it can be confusing otherwise... Maybe I'm missing some detail that would make that too difficult though.

Reply:
no detail besides just wondering if it's worth the API change... i do agree offeringId is clearer...